packages/integration-tests/src/test-e2e/deployment.test.ts

```diff
@@ -107,6 +108,14 @@ void describe('deployment tests', { concurrency: testConcurrencyLevel }, () => {
               format
             );
           }
+
+          // test deployed backend client outputs
+          await execa('node', ['verify_outputs.js'], {
```

Comment:
I wonder if this should be a method on testProject since it is tightly coupled to the project setup?

Reply:
I don't see why not, I'll add a new method